Homework10/Homework10/Services/ElectricalApplianceService.cs

```diff
@@ -0,0 +1,182 @@
+                            refrigerator.Name,
+                            refrigerator.IsPluggedIn,
+							refrigerator.TemperatureInside
+                            );
+                    case TelevisionEntity television:
+                        return new Television(
+                            television.PowerConsumption,
+                            television.Name,
+                            television.IsPluggedIn,
+                            television.ScreenDiagonal
+                        );
+                    case WashingMachineEntity washingMachine:
+                        return new WashingMachine(
+                            washingMachine.PowerConsumption,
+                            washingMachine.Name,
+                            washingMachine.IsPluggedIn,
+                            washingMachine.NumberOfPrograms
+                        );
+                    case DishwasherEntity dishwasher:
+                        return new Dishwasher(
+                            dishwasher.PowerConsumption,
+                            dishwasher.Name,
+                            dishwasher.IsPluggedIn,
+                            dishwasher.NumberOfPrograms,
+                            dishwasher.CapacityOfCookwareSets
+                        );
+                    case MicrowaveEntity microwave:
+                        return new Microwave(
+                            microwave.PowerConsumption,
+                            microwave.Name,
+                            microwave.IsPluggedIn,
+                            microwave.Capacity
+                        );
+                    case KettleEntity kettle:
+                        return new Kettle(
+                            kettle.PowerConsumption,
+                            kettle.Name,
+                            kettle.IsPluggedIn,
+                            kettle.Capacity
+                        );
+                    case IronEntity iron:
+                        return new Iron(
+                            iron.PowerConsumption,
+                            iron.Name,
+                            iron.IsPluggedIn,
+                            iron.CordLength
+                        );
+                    default:
+                        return null;
+
+                }
+
+            }
+
+            return null;
+        }
+        
+
+        public ElectricalAppliance[] GetElectricalAppliances()
+        {
+            var electricalAppliancesLength = _electricalAppliancesRepository.GenerateElectricalAppliance().Length;
+            ElectricalAppliance[] electricalAppliances = new ElectricalAppliance[electricalAppliancesLength];
+
+            for (int i = 0; i < electricalAppliances.Length; i++)
+            {
+                electricalAppliances[i] = GetElectricalApplianceByNumber(i);
+            }
+
+            return electricalAppliances;
+        }
+
+        public ElectricalAppliance ChooseElectricalAppliance()
```

Comment:
It's better to work with the console as high as possible; therefore, it would be better to use Console only in Startup